fusillade/clouddirectory.py

```diff
@@ -0,0 +1,788 @@
+import functools
+import json
+import typing
+import boto3
+from fusillade.errors import FusilladeException
+from collections import namedtuple
+from enum import Enum, auto
+from urllib.parse import quote, unquote
+import os
+
+ad = boto3.client("clouddirectory")
+arn = "arn:aws:clouddirectory:us-east-1:861229788715:"
```

Comment:
I could easily see someone in the future writing a function using the var `arn` below and getting unexpected behavior if it was undeclared and yet returned a value where unexpected.  Consider a more unique name and/or all caps for var declarations at the top of a file?

Reply:
changed to project_arn